CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## `4.0.0`
```

Comment:
4? This is a breaking change?

Reply:
Not sure if breaking change, but it does contain changes in the behavior. Specially for people using `primaryKey` or people relying on the exception raised for `url` not being defined. Better safe than sorry :)